gui/control_main.py

```diff
@@ -3718,8 +3720,10 @@ def addRequestsToAllSelectedCB(self):
                 elif itemDataType == "request":
                     selectedSampleRequest = db_lib.getRequestByID(item.data(32))
                     self.selectedSampleID = selectedSampleRequest["sample"]
-                
-                if self.selectedSampleID in samplesConsidered: # If a request is already added to the sample, move on
+
```

Comment:
can you remove this code change? I know it looks like an obvious linting fix, but shouldn't be here

Reply:
Removed in 7a3c09728df0e70791374f409280d708f8fcc6e5